src/toast/ops/cadence_map.py

```diff
@@ -154,7 +155,9 @@ def _exec(self, data, detectors=None, **kwargs):
             buf[:, :] = False
             for obs in data.obs:
                 obs_data = data.select(obs_uid=obs.uid)
-                dets = obs.select_local_detectors(detectors)
+                dets = obs.select_local_detectors(
+                    detectors, flagmask=self.det_flag_mask
```

Comment:
That is an interesting point- for our default simple cases it is fine, since we typically have "invalid" or "nonscience" and nonscience is a superset that includes the invalid bit.  However I guess we should support different masks for per-detector and per-sample flags.  Maybe `det_mask` and `det_flag_mask`?

Reply:
This is now implemented throughout the code base.  The `det_mask` trait describes the bit mask applied to per-detector flags and the `det_flag_mask` applies to per-sample flags.